cmd/manager/main.go

```diff
@@ -83,31 +87,42 @@ func main() {
 	}
 
 	// Get a config to talk to the apiserver
-	clientConfig, err := config.GetConfig()
+	cfg, err := config.GetConfig()
```

Comment:
I renamed it back to `cfg` just to minimize deviation from the code generated by operator-sdk.